server/server.js

```diff
@@ -13,8 +13,8 @@ const pool = new Pool({
     "postgres://user:BzA52BUb5uHl8B2oNiSWM8uixzot4m7W@dpg-cfakkd82i3mjduj6onq0-a.frankfurt-postgres.render.com/dbvideos",
   ssl: { rejectUnauthorized: false },
   user: "user",
-  // host: "postgres://user:BzA52BUb5uHl8B2oNiSWM8uixzot4m7W@dpg-cfakkd82i3mjduj6onq0-a/dbvideos",
-  host: "postgres://user:BzA52BUb5uHl8B2oNiSWM8uixzot4m7W@dpg-cfakkd82i3mjduj6onq0-a.frankfurt-postgres.render.com/dbvideos", //external
+  host: "postgres://user:BzA52BUb5uHl8B2oNiSWM8uixzot4m7W@dpg-cfakkd82i3mjduj6onq0-a/dbvideos",
+  // host: "postgres://user:BzA52BUb5uHl8B2oNiSWM8uixzot4m7W@dpg-cfakkd82i3mjduj6onq0-a.frankfurt-postgres.render.com/dbvideos", //external
   database: "dbvideos",
   password: "BzA52BUb5uHl8B2oNiSWM8uixzot4m7W",
```

Comment:
👋 This is a really common junior mistake so it's good you've made it now! 😓 😄 

Never commit secrets to your codebase. This is not secure. I've got your DB password now! Instead, put your passwords in an .env file that is never committed. .env means Environment and you might have different ones for production and development, or dev and staging. 

https://create-react-app.dev/docs/adding-custom-environment-variables/